dotnet/src/SemanticKernel/Planning/Plan.cs

```diff
@@ -166,7 +166,7 @@ public Plan(ISKFunction function)
     /// <remarks>If Context is not supplied, plan will not be able to execute.</remarks>
     public static Plan FromJson(string json, SKContext? context = null)
     {
-        var plan = JsonSerializer.Deserialize<Plan>(json, new JsonSerializerOptions() { IncludeFields = true }) ?? new Plan(string.Empty);
+        var plan = JsonSerializer.Deserialize<Plan>(json, new JsonSerializerOptions { IncludeFields = true }) ?? new Plan(string.Empty);
```

Comment:
nit: could probably just be `new`, without the JsonSerializerOptions 
...but I don't mind having it show what kind it is explicitly.